base/sim/fastsim/FairFastSimModel.cxx

```diff
@@ -97,7 +97,7 @@ void FairFastSimModel::DoIt(const G4FastTrack& fastTrack, G4FastStep& fastStep)
             particle->GetPolarisation(polVect);
             G4ThreeVector pol(polVect.X(), polVect.Y(), polVect.Z());   // get polarisation
             G4double tim = particle->T() * 10.e9;                       // change from ns to s
-            G4double len = 0.;
+                                                                        //   G4double len = 0.;
```

Comment:
If `len` is unused, better remove it. The next reader of the code will be rather puzzled, trying to understand the comment. :wink:

Reply:
removed in #979